### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by StoneWolf**
Fix typos/grammar.

### CODE REVIEW

File: input/docs/getting-started/index.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Let's create a simple application demonstrating a number of ReactiveUI functiona
 We will create a WPF application, which will allow us to search through Flickr public images.   
 The full code of the application is shown at the end of this chapter, and we will show relevant snippets as we go.
 
-In Visual Studio create a new WPF application (.Net 4.5 or above)
+In Visual Studio, create a new WPF application (.NET 4.5 or above).
```
Comment (glennawatson) [inline input/docs/getting-started/index.md]: I believe we increased that to like 4.6.1 now.

Comment (glennawatson) [reply input/docs/getting-started/index.md]: Looks good otherwise.

Comment (StoneWolf) [reply input/docs/getting-started/index.md]: https://github.com/reactiveui/ReactiveUI/blob/f2290c290f189ab349dcdae0a5cc752a69fb06f7/src/ReactiveUI.Wpf/ReactiveUI.Wpf.csproj#L9

Seems like it!

**Comment by glennawatson** [comment]
@onovotny Is there an issue with that builder? Eg the website PR validation task failed with a time out.

**Comment by onovotny** [comment]
Two things: 

1. I don't see a vsts yaml file checked in anywere...it really should be for the build
2. The publish step needs a condition `and(succeeded(), eq(variables['system.pullrequest.isfork'], false))` because forks cannot publish artifacts currently.

**Pull request merged**